package.json

```diff
@@ -93,6 +93,7 @@
     "vega": "^5.30.0",
     "vega-lite": "^5.21.0"
   },
+  "packageManager": "npm@10.8.2",
```

Comment:
Why is this needed?

Reply:
[corepack](https://github.com/nodejs/corepack) is a good standard when working with a lot of different package managers on a lot of different projects. 

This field is the standard field that a lot of tools (including corepack) will use to determine the package manager.

I can remove it if you want

Reply:
I just don't feel like we specifically need that version of npm. Yeah, let's remove it since we don't consistently use it in this org.